guides/source/5_0_release_notes.md

```diff
@@ -227,6 +227,10 @@ Please refer to the [Changelog][action-pack] for detailed changes.
     `RedirectBackError`.
     ([Pull Request](https://github.com/rails/rails/pull/22506))
 
+*   `ActionDispatch::IntegrationTest` and
+    `ActionController::TestCase` accept keyword arguments instead of
+positional arguments. ([Pull Request](https://github.com/rails/rails/pull/18323))
```

Comment:
What's off about the alignment here? MacVim seems to be trying to imitate the alignment of the file, and appears to render ok on GitHub, as well:
<img width="860" alt="screen shot 2016-04-20 at 4 32 57 pm" src="https://cloud.githubusercontent.com/assets/3020626/14689503/99f99b7c-0715-11e6-908c-8b33c2c4d7b3.png">